README.md

```diff
@@ -108,120 +102,74 @@ There are several demonstrations of common cases.
-  export KUBECONFIG="/root/.kube/karmada.config"
-Or use kubectl with --kubeconfig=/root/.kube/karmada.config
-Please use 'kubectl config use-context <Context_Name>' to switch clusters.
-The following is context intro:
-  ------------------------------------------------------
-  |    Context Name   |          Purpose               |
-  |----------------------------------------------------|
-  | karmada-host      | the cluster karmada install in |
-  |----------------------------------------------------|
-  | karmada-apiserver | karmada control plane          |
-  ------------------------------------------------------
+To start using your karmada, run:
+  export KUBECONFIG=/root/.kube/karmada.config
+Please use 'kubectl config use-context karmada-host/karmada-apiserver' to switch the host and control plane cluster.
+
+To manage your member clusters, run:
+  export KUBECONFIG=/root/.kube/members.config
+Please use 'kubectl config use-context member1/member2/member3' to switch to the different member cluster.
 ```
 
-There are two contexts:
+There are two contexts about karmada:
 - karmada-apiserver `kubectl config use-context karmada-apiserver`
 - karmada-host `kubectl config use-context karmada-host`
 
 The `karmada-apiserver` is the **main kubeconfig** to be used when interacting with karamda control plane, while `karmada-host` is only used for debugging karmada installation with the host cluster. You can check all clusters at any time by running: `kubectl config view`. To switch cluster contexts, run `kubectl config use-context [CONTEXT_NAME]`
 
-#### Tips
-- Please make sure you can access google cloud registry: k8s.gcr.io
-- Install script will download golang package, if your server is in the mainland China, you may set go proxy like this `export GOPROXY=https://goproxy.cn`
-
-### Join member cluster
-In the following steps, we are going to create a member cluster and then join the cluster to
-karmada control plane.
-
-#### 1. Create member cluster
-We are going to create a cluster named `member1` and we want the `KUBECONFIG` file
-in `$HOME/.kube/karmada.config`. Run following command:
-```
-hack/create-cluster.sh member1 $HOME/.kube/karmada.config
-```
-The script `hack/create-cluster.sh` will create a cluster by kind.
-
-#### 2. Join member cluster to karmada control plane
-
-You can choose one of mode: [push](#21-push-mode-karmada-controls-the-member-cluster-initiative-by-using-karmadactl) or
-[pull](#22-pull-mode-installing-karmada-agent-in-the-member-cluster), either will help you join a member cluster.
-
-##### 2.1. Push Mode: Karmada controls the member cluster initiative by using `karmadactl`
+#### Network resource requirements:
+Please make sure you can access the following resources, any network problem please refer to the [troubleshooting](/docs/troubleshooting.md#i-cant-access-some-resources-when-install-karmada).
+- Google Container Registry: k8s.gcr.io
+- Download golang package, such as golang.org, k8s.io, etc.
```

Comment:
```suggestion
```
It's too late to tell users the network requirements, they might have met the problem before reading these tips.

We might add a summary description at `Prerequisites`. (No further suggestion, for now, we can add it later.)